src/containers.js

```diff
@@ -15,6 +17,8 @@ const validComposeFile = async (contents) => {
   return valid;
 };
 
+const backupIsEnabled = () => process.env.CHT_BACKUP_COMPOSE_FILES !== 'false';
```

Comment:
`isBackupEnabled = () => ` 
I think verb goes first 🤔

Just as you did with `hasBackupDir` from the utils file